src/main/java/org/gradle/wrapperupgrade/UpgradeWrapper.java

```diff
@@ -201,6 +203,36 @@ private void gitCreatePr(Params params, String prTitle, String prBody) throws IO
             if (!labels.isEmpty()) {
                 pr.addLabels(labels.toArray(new String[0]));
             }
+            List<String> reviewers = upgrade.getOptions().getReviewers().get();
+            if (!reviewers.isEmpty()) {
+                List<GHUser> githubReviewers = reviewers.stream()
+                    .map(reviewer -> {
+                        try {
+                            return params.gitHub.getUser(reviewer);
+                        } catch (IOException e) {
+                            getLogger().warn(String.format("Error fetching GitHub user '%s'", reviewer), e);
+                            return null;
+                        }
```

Comment:
Suggesting to extract as a function and re-use for the assignees

Reply:
Done here https://github.com/gradle/wrapper-upgrade-gradle-plugin/pull/271/commits/d4cbfb935661209acf1961d03a742b1348e10798